plugins/wazo-snom/10.1.101.11/pkgs/pkgs.db

```diff
@@ -75,60 +75,73 @@ version: 2.1.1
 files: uxm-fw
 install: snom-fw
 
+[pkg_uxmc-fw]
+description: Firmware for Snom Extension USB Module UXMC D7C
+description_fr: Micrologiciel pour module d'extension USB Snom UXMC D7C
+version: 1.1.1
+files: uxmc-fw
+install: snom-fw
+
 [install_snom-fw]
 a-b: cp $FILE1 firmware/
+b-c: touch -c firmware/$FILE1
```

Comment:
Mmmh seems like this makes the plugin fails to install (see last line of the stacktrace below) :
```
2022-07-04 16:54:29,721 [28336] (ERROR) (provd.app): Error while loading plugin wazo-snom-10.1.101.11
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/provd/app.py", line 824, in _pg_load
    self.pg_mgr.load(id, gen_cfg, spec_cfg)
  File "/usr/lib/python2.7/dist-packages/provd/plugins.py", line 1134, in load
    plugin = obj(self._app, plugin_dir, gen_cfg, spec_cfg)
  File "/var/lib/wazo-provd/plugins/wazo-snom-10.1.101.11/common.py", line 153, in __init__
    fetchfw_helper = FetchfwPluginHelper(plugin_dir, downloaders)
  File "/usr/lib/python2.7/dist-packages/provd/plugins.py", line 554, in __init__
    self._pkg_mgr = self._new_pkg_mgr(downloaders, filter_builder)
  File "/usr/lib/python2.7/dist-packages/provd/plugins.py", line 572, in _new_pkg_mgr
    pkg_builder)
  File "/usr/lib/python2.7/dist-packages/xivo_fetchfw/storage.py", line 386, in __init__
    self._load_pkgs()
  File "/usr/lib/python2.7/dist-packages/xivo_fetchfw/storage.py", line 393, in _load_pkgs
    self._pkgs = self._create_pkg(config, pkg_sections, remote_files, install_mgr_factories)
  File "/usr/lib/python2.7/dist-packages/xivo_fetchfw/storage.py", line 455, in _create_pkg
    pkg = self._pkg_builder.build_installable_pkg(config, section, pkg_id, remote_files, install_mgr_factories)
  File "/usr/lib/python2.7/dist-packages/xivo_fetchfw/storage.py", line 280, in build_installable_pkg
    pkg_install_mgr = install_mgr_factory.new_install_mgr(src_node, local_vars)
  File "/usr/lib/python2.7/dist-packages/xivo_fetchfw/storage.py", line 188, in new_install_mgr
    filter_obj = self._filter_builder.build_node(tokens)
  File "/usr/lib/python2.7/dist-packages/xivo_fetchfw/storage.py", line 144, in build_node
    raise ValueError("unknown node type %s" % type)
ValueError: unknown node type touch
```

Did I miss something to execute the `touch` system command ?
Thx 👍